scripts/setup_haskell.hs

```diff
@@ -352,3 +332,94 @@ filePathToText = Turtle.format Turtle.fp
 
 textToFilePath :: Text -> FilePath
 textToFilePath = FS.fromText
+
+hvn :: Text
+hvn = "haskell-vim-now"
+
+type HelperRepository = Text
+type HelperTool = Text
+type Resolver = Text
+
+-- install ghcide, write compiler and file type plugin files
+installGhcide :: (MonadIO m, MonadReader HvnConfig m) => m ()
+installGhcide = do
+  hvnCfgDest' <- asks hvnCfgDest
+  gitCloneInstall "https://github.com/digital-asset/ghcide.git" "ghcide" Nothing
```

Comment:
Does ghcide need to be installed with a specific resolver?  If the user had some older LTS specified in their global stack config, the install could fail.  I could also see pushing that to the user to address though.

Reply:
It seems [ghcide](https://github.com/digital-asset/ghcide) supports GHC >= 8.4 only. I'm afraid it wouldn't compile with older LTS.